src/modals/Modal.tsx

```diff
@@ -49,17 +49,25 @@ export class Modal extends React.PureComponent<ModalProps> {
 
     componentWillUpdate(nextProps: ModalProps, nextState: ModalState) {
         if (this.isOpen(nextProps, nextState)) {
-            if (this.divRef === null) {
-                const el = document.createElement("div");
-                document.body.appendChild(el);
-                this.divRef = el;
-            }
-            document.body.classList.add(ClassNames.NO_SCROLLING);
+            this.createDivRef();
         } else {
             this.cleanup();
         }
     }
 
+    componentWillMount() {
+        this.createDivRef();
```

Comment:
Why do we have to create the placeholder <div/> here?  Can't we just rely on componentWillUpdate to create the <div/> placeholder?  I commented out this function and everything seems to work fine.

Reply:
I was facing an issue where we navigate to another page from the modal page with an opened modal and move back to the modal page I get the error "Target  container is not a DOM element". Arpita had already resolved this issue using componentWillMount().

![image](https://user-images.githubusercontent.com/10673883/61956373-9c359780-afda-11e9-9f7a-f5fac4fc5bff.png)

Reply:
Okay, got it.  Thanks.